### DISCUSSION TITLE
disableAnimation warning still on 2.4.6

### DISCUSSION THREAD

**Discussion by uigywnkiub** [heroui-inc/heroui#3520] [Feedback]
Warning: React does not recognize the `disableAnimation` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `disableanimation` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at img...

**Comment by wingkwong** [comment]
I couldn't reproduce the issue. can you share a reproducible code?

**Reply by uigywnkiub** [reply]
> I couldn't reproduce the issue. can you share a reproducible code?

https://github.com/nextui-org/nextui/issues/3521

**Reply by wingkwong** [reply]
I couldn't reproduce the issue using his code in v2.4.6. Besides, he's using v2.4.1.

**Reply by uigywnkiub** [reply]
> I couldn't reproduce the issue using his code in v2.4.6. Besides, he's using v2.4.1.

In my case it still bug on 2.4.6, maybe I need to delete .next and node_model use folder?

**Reply by wingkwong** [reply]
are you using `latest` tag? you may try removing those directories and reinstall again. if it still fails, please share your code for us to investigate. thanks.

**Reply by uigywnkiub** [reply]
> are you using `latest` tag? you may try removing those directories and reinstall again. if it still fails, please share your code for us to investigate. thanks.

Latest yeap, I reinstalled, bug still.

**Reply by uigywnkiub** [reply]
Reproduce code:
```js
1)

import { getAuthSession } from '@/app/lib/actions'

import UserProfileInfo from './user-profile-info'

export default async function User() {
  const session = await getAuthSession()

  return <UserProfileInfo user={session?.user} />
}
```

```js
2)
'use client'

import Image from 'next/image'

import { TDefaultSession } from '@/app/lib/types'
import { User } from '@nextui-org/react'

type TProps = {
  user: TDefaultSession['user']
}

function UserProfileInfo({ user }: TProps) {
  return (
    <User
      name={user?.name}
      description={user?.email}
      avatarProps={{
        ImgComponent: Image,
        imgProps: {
          // @ts-ignore
          priority: true,
          quality: 100,
          width: 40,
          height: 40,
        },
        src: user?.image || '',
      }}
    />
  )
}

export default UserProfileInfo
```

Error in console browser and node - 
<img width="623" alt="image" src="https://github.com/user-attachments/assets/dd7c9bda-a801-4596-96f6-fb858efe2daa">
